packages/server/assets/core/typedefs/user.graphql

```diff
@@ -42,7 +42,14 @@ extend type Query {
     cursor: String
     archived: Boolean = false
     emailOnly: Boolean = false
-  ): UserSearchResultCollection!
+  ): UserSearchResultCollection! @deprecated(reason: "Use users() instead.")
+
+  """
+  Look up server users
+  """
+  users(input: UsersRetrievalInput!): UserSearchResultCollection!
```

Comment:
The param list for `userSearch` was already too long and I expect it to grow in the future. Plus, why would we want to allow (non-admin)users to search for archived users?